src/github.com/getlantern/flashlight/ops/ops.go

```diff
@@ -27,12 +27,12 @@ type Op struct {
 }
 
 // Enter mimics the similar method from ops.Op
-func (op *Op) Enter(name string) *Op {
+func (op *Op) Begin(name string) *Op {
```

Comment:
@fffw You convinced me.  I've applied the changes here in flashlight, and I'll apply them to the core ops library next.